package-lock.json

```diff
@@ -1,14566 +1,8 @@
 {
   "name": "loast",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
`package-lock.json` should stay version 2

Reply:
I totally misread that. Thanks